docs/windows.md

```diff
@@ -184,13 +184,6 @@ spec:
       hostNetwork: true
 ```
 
-If the powershell version on your Window Node is earlier than v6.0,
-change the command from `pwsh` to `powershell` in kube-proxy.yml.
-
-```bash
-sed -i 's/pwsh/powershell/g' kube-proxy.yml
-```
-
```

Comment:
@GraysonWu you added this a couple of months ago to address https://github.com/antrea-io/antrea/issues/3245. However, I find this confusing because it's not the Powershell version on the Node that matters, since the command is run inside a container. For example, the Powershell version on my Windows Node is v5, and I do not need that change. Furthermore, I tested all the `kube-proxy:<VERSION>-nanoserver` with `VERSION >= v1.18`, and they all have `pwsh` available. Only legacy images (which don't use nanoserver) are missing `pwsh`, but https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/kube-proxy.yml doesn't use these legacy images.

```
PS C:\k\antrea> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.17763.2268
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.2268
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1


PS C:\k\antrea> docker run sigwindowstools/kube-proxy:v1.18.0 pwsh
docker: Error response from daemon: container 09ca691e4f9fde8c44c820ab3f523ce4fc0c0489ed79fecd3cff5bd74a25b402 encountered an error during hcsshim::System::CreateProcess: failure in a Windows system call: The system cannot find the file specified. (0x2).
PS C:\k\antrea> docker run sigwindowstools/kube-proxy:v1.18.0-nanoserver pwsh
PowerShell 7.2.2
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

PS C:\>
PS C:\k\antrea>
```

Reply:
You are right. It could be removed.

Reply:
Thanks for pointing this out. I just noticed that somehow the Windows testbed I have are still using the "legacy images (which don't use nanoserver)". I got confused when I was trying to solve issue #3245.